### ISSUE TITLE
Suggestion: Only add/remove points when window is in focus

### ISSUE THREAD

**Issue by junglist** [HabitRPG/habitica#636]
I couldn't find this suggestion in trello or here, so I'm sorry if it's already added somewhere.

I spend all day at work browsing, I often keep a separate desktop open with a browser tab on Facebook, but I'm not on it all day :-)

I do periodically check Facebook but leave the tab open all day. However, the chrome extension seems to record my activity regardless of whether or not I'm active in the window.

I know from other apps that it's possible to track the tab/window activity so I'm wondering if it's possible to also include this with the extension.

Basically that means that you only gain/lose xp/health if you are viewing the page, rather than just having it open in a background tab. This has actually caused me to die a few times  because I forget the tab is open.

Keep up the good work!

**Comment by lefnire** [comment]
Absolutely, very important request. Follow up here. https://github.com/lefnire/habitrpg-chrome/issues/52
